### PR TITLE
Adding a new 'Get Started' section on the homepage

### PR DESCRIPTION
**Pull request by AleemRehman**
Adding the new section requested in issue: https://github.com/SimplQ/simplQ-frontend/issues/416
This includes adding a similar call-to-action section on the homepage which should up conversions:
image

**This is currently a WIP waiting for feedback on the issue about which buttons are required!**

### CODE REVIEW

File: simplq/src/components/pages/Home/StaticInfos.jsx [hunk 1]
```diff
@@ -124,3 +124,9 @@ export const ExtraInfo = () => (
     </div>
   </div>
 );
+
+export const GetStarted = () => (
+  <div data-aos="fade-up" className={`${styles['section']} ${styles['get-started']}`}>
+    <h2 data-aos="zoom-in">Ready to get started with SimplQ?</h2>
```
Comment (daltonfury42) [inline simplq/src/components/pages/Home/StaticInfos.jsx:130]: Can you add a button that on click will scroll all the way to the top? You can import and use this function: 

https://github.com/SimplQ/simplQ-frontend/blob/master/simplq/src/components/common/utilFns.js#L24

**Comment by AleemRehman** [comment]
Those changes have been made @daltonfury42!

**Comment by daltonfury42** [comment]
Awesome, can you add a screenshot of how it will look like, to the PR? Also, can you make sure it looks fine on mobile as well?

**Comment by AleemRehman** [comment]
Desktop view:
image


Mobile view:
image

**Review by daltonfury42 — APPROVED** [review]
Thanks for the PR. :)

**Pull request merged**